packages/resolver/lib/sources/globalnpm.ts

```diff
@@ -44,7 +44,7 @@ export class GlobalNPM implements ResolverSource {
       try {
         const result = fs.readFileSync(possiblePath, "utf8");
         return JSON.parse(result);
-      } catch (e) {
+      } catch (_err) {
```

Comment:
Style note: You can actually just do `catch { ... }` if you're not using the error object.

Reply:
I didn't know that! thanks @haltman-at . Updated.